joblib/memory.py

```diff
@@ -882,7 +882,13 @@ class Memory(Logger):
             as functions are evaluated.
 
         bytes_limit: int, optional
-            Limit in bytes of the size of the cache.
+            Limit in bytes of the size of the cache. By default, the size of
+            the cache is unlimited.
+
+            .. note::
+
+               You need to call :meth:`~joblib.Memory.reduce_size` to actually
```

Comment:
Maybe it would be clearer to not hide the fact that `reduce_size` is a method of `Memory`?

```suggestion
               You need to call :meth:`joblib.Memory.reduce_size` to actually
```